From df227481504574ecc6028400d17870913a16a047 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Thu, 23 Jan 2020 16:41:26 +0200
Subject: [PATCH 2/3] rtsp-latency-bin: replace G_TYPE_INSTANCE_GET_PRIVATE as
 it's been deprecated

from glib
```
Deprecated: 2.58: Use %G_ADD_PRIVATE and the generated
  `your_type_get_instance_private()` function instead
```
---
 gst/rtsp-server/rtsp-latency-bin.c | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/gst/rtsp-server/rtsp-latency-bin.c b/gst/rtsp-server/rtsp-latency-bin.c
index cf7cdf1..c297ab6 100644
--- a/gst/rtsp-server/rtsp-latency-bin.c
+++ b/gst/rtsp-server/rtsp-latency-bin.c
@@ -23,9 +23,6 @@
 #include <gst/gst.h>
 #include "rtsp-latency-bin.h"
 
-#define GST_RTSP_LATENCY_BIN_GET_PRIVATE(obj)  \
-     (G_TYPE_INSTANCE_GET_PRIVATE ((obj), GST_RTSP_LATENCY_BIN_TYPE, GstRTSPLatencyBinPrivate))
-
 struct _GstRTSPLatencyBinPrivate
 {
   GstPad *sinkpad;
@@ -106,7 +103,7 @@ gst_rtsp_latency_bin_get_property (GObject * object, guint propid,
 {
   GstRTSPLatencyBin *latency_bin = GST_RTSP_LATENCY_BIN (object);
   GstRTSPLatencyBinPrivate *priv =
-      GST_RTSP_LATENCY_BIN_GET_PRIVATE (latency_bin);
+      gst_rtsp_latency_bin_get_instance_private (latency_bin);
 
   switch (propid) {
     case PROP_ELEMENT:
@@ -140,7 +137,7 @@ gst_rtsp_latency_bin_add_element (GstRTSPLatencyBin * latency_bin,
     GstElement * element)
 {
   GstRTSPLatencyBinPrivate *priv =
-      GST_RTSP_LATENCY_BIN_GET_PRIVATE (latency_bin);
+      gst_rtsp_latency_bin_get_instance_private (latency_bin);
   GstPad *pad;
   GstPadTemplate *templ;
 
@@ -250,7 +247,7 @@ static gboolean
 gst_rtsp_latency_bin_recalculate_latency (GstRTSPLatencyBin * latency_bin)
 {
   GstRTSPLatencyBinPrivate *priv =
-      GST_RTSP_LATENCY_BIN_GET_PRIVATE (latency_bin);
+      gst_rtsp_latency_bin_get_instance_private (latency_bin);
   GstEvent *latency;
   GstQuery *query;
   GstClockTime min_latency;
-- 
2.20.1

